### PR TITLE
Gramine mrenclave

### PR DESCRIPTION
**Pull request by PierreJeanjacquot**
- update gramine mrenclave format
- update nodejs required version
- format edited files

### CODE REVIEW

File: for-developers/confidential-computing/create-your-first-gramine-app.md [hunk 1]
```diff
@@ -222,14 +219,14 @@ iexec task debug <taskid> --logs --chain bellecour
...
-| `9984` | `Invalid SPS Certificate` | The SSL certificate of the SPS is not signed by a Certificate Authority you trust. | You might be using a `<version>` of the Gramine base (`iexechub/iexec-gramine-base:<version>`) which is too old. Verify the `<version>` to use [in the documentation](#prepare-your-application) or please contact [iExec Help Center](https://iexecproject.atlassian.net/servicedesk/customer/portal/4). |
-| `30952` | `Unexpected MRENCLAVE` | The measurement of the enclave does not match your on-chain configuration of your deployed dapp `"fingerprint": "<mrenclave>"`. | It is likely you did not set the `<mrenclave>` properly, please verify [how to retrieve it](#deploy-the-tee-app-on-iexec). |
+| Error code | Error message             | Description                                                                                                                     | Action                                                                                                                                                                                                                                                                                                    |
```
Comment (james-toussaint) [inline for-developers/confidential-computing/create-your-first-gramine-app.md]: Is it added by the linter?

File: for-developers/confidential-computing/create-your-first-gramine-app.md [hunk 2]
```diff
@@ -10,10 +10,10 @@ Before going any further, make sure you managed to [Build your first application
 **Prerequisites**
 
 - [Docker](https://docs.docker.com/install/) 17.05 or higher on the daemon and client.
-- [Nodejs](https://nodejs.org) 14.0.0 or higher.
+- [Nodejs](https://nodejs.org) 14.17.1 or higher.
 - [iExec SDK](https://www.npmjs.com/package/iexec) 8.0.0 or higher.
 - Familiarity with the basic concepts of [Intel® SGX](intel-sgx-technology.md#intel-r-software-guard-extension-intel-r-sgx) and [Gramine](choose-your-tee-framework.md#gramine) framework.
-{% endhint %}
+  {% endhint %}
```
Comment (james-toussaint) [inline for-developers/confidential-computing/create-your-first-gramine-app.md]: I'm not sure about this lint by looking to the first `{% hint style="success" %}`

Comment (PierreJeanjacquot) [reply for-developers/confidential-computing/create-your-first-gramine-app.md]: yes it look wierd

File: for-developers/confidential-computing/create-your-first-gramine-app.md (hunk 1 above)
Comment (PierreJeanjacquot) [reply for-developers/confidential-computing/create-your-first-gramine-app.md]: added by `prettier` makes tables human-readable without rendering

Comment (james-toussaint) [reply for-developers/confidential-computing/create-your-first-gramine-app.md]: I suggest for tables to stick with the compact format https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables#formatting-content-within-your-table which is more maintainable and results with less diffs when content will be updated :)

File: for-developers/confidential-computing/create-your-first-gramine-app.md (hunk 2 above)
Comment (PierreJeanjacquot) [reply for-developers/confidential-computing/create-your-first-gramine-app.md]: fixed 1c2638776b3aa761ad2bbc21c835c9bd61b7e909
it was interpreted as being part of the unordered list

Comment (james-toussaint) [reply for-developers/confidential-computing/create-your-first-gramine-app.md]: Nice thanks!

File: for-developers/confidential-computing/create-your-first-gramine-app.md (hunk 1 above)
Comment (PierreJeanjacquot) [reply for-developers/confidential-computing/create-your-first-gramine-app.md]: reformated with `"proseWrap": "never"` c3eedb5317a560a1dc27a09212a09ae910b9c775

this rule:
- format tables with compact syntax
- replace singles End Of Line by space (MD renders single EOL as space, double EOL are rendered as EOL)

File: for-developers/your-first-app.md [hunk 3]
```diff
@@ -216,12 +215,12 @@ The runtime variables are environment variables set by the iExec worker and avai
...
 | Name | Type | Content |
-|---|---|---|
+| --- | --- | --- |
 | IEXEC_OUT | path | Absolute path of iexec output folder |
 | IEXEC_TASK_ID | bytes32 | taskid of the running task |
-| IEXEC_BOT_TASK_INDEX  | int &gt;= 0 | Index of the current task in the Bag of Tasks* |
-| IEXEC_BOT_FIRST_INDEX | int &gt;= 0 | Index of the first task in the current Deal \(Bag of task* subset\) |
-| IEXEC_BOT_SIZE | int &gt;= 1 | Total number of parallelized tasks in a Bag of Tasks* |
+| IEXEC_BOT_TASK_INDEX | int &gt;= 0 | Index of the current task in the Bag of Tasks\* |
+| IEXEC_BOT_FIRST_INDEX | int &gt;= 0 | Index of the first task in the current Deal \(Bag of task\* subset\) |
```
Comment (james-toussaint) [inline for-developers/your-first-app.md:222]:
```suggestion
| IEXEC_BOT_FIRST_INDEX | int &gt;= 0 | Index of the first task in the current Deal (Bag of task\* subset) |
```

File: for-developers/your-first-app.md [hunk 4]
```diff
@@ -216,12 +215,12 @@ The runtime variables are environment variables set by the iExec worker and avai
...
 
 | Name | Type | Content |
-|---|---|---|
+| --- | --- | --- |
 | IEXEC_OUT | path | Absolute path of iexec output folder |
 | IEXEC_TASK_ID | bytes32 | taskid of the running task |
-| IEXEC_BOT_TASK_INDEX  | int &gt;= 0 | Index of the current task in the Bag of Tasks* |
-| IEXEC_BOT_FIRST_INDEX | int &gt;= 0 | Index of the first task in the current Deal \(Bag of task* subset\) |
-| IEXEC_BOT_SIZE | int &gt;= 1 | Total number of parallelized tasks in a Bag of Tasks* |
+| IEXEC_BOT_TASK_INDEX | int &gt;= 0 | Index of the current task in the Bag of Tasks\* |
```
Comment (james-toussaint) [inline for-developers/your-first-app.md:221]: Safer with backslash I guess?

File: for-developers/quick-start-for-developers.md [hunk 5]
```diff
@@ -122,7 +131,7 @@ iexec app init
 The iExec SDK writes the minimum app configuration in `iexec.json`
 
 | **key** | **description** |
-| :--- | :--- |
+| :-- | :-- |
```
Comment (james-toussaint) [inline for-developers/quick-start-for-developers.md:134]:
```suggestion
| --- | --- |
```

File: for-developers/quick-start-for-developers.md [hunk 6]
```diff
@@ -122,7 +131,7 @@ iexec app init
...
 | owner | app owner ethereum address \(default your wallet address\) |
```
Comment (james-toussaint) [inline for-developers/quick-start-for-developers.md:135]: Backslashes seem useless :) (But PR is already big, so maybe later)

**Review by james-toussaint — APPROVED** [review]

**Pull request merged**